Basic/ClientDriven/Assets/Scripts/NetworkObjectSpawner.cs

```diff
@@ -0,0 +1,46 @@
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.Networking;
+using Random = System.Random;
+
+namespace Unity.Multiplayer.Samples.ClientDriven
+{
+    /// <summary>
+    /// Spawn a NetworkObject at this transform's position when NetworkManager's server is started.
+    /// </summary>
+    /// <remarks>
+    /// A NetworkManager is expected to be part of the scene that this NetworkObject is a part of.
+    /// </remarks>
+    internal class NetworkObjectSpawner : MonoBehaviour
+    {
+        [SerializeField]
+        NetworkObject m_PrefabReference;
+
+        void Start()
+        {
+            Debug.Assert(NetworkManager.Singleton != null, "The NetworkManager needs to be referenced!");
```

Comment:
Contextually, this doesn't make sense since we don't set it. You could signal that a NetworkManager is likely not part of this MonoBehaviour's scene, or along those lines.

Reply:
Yes I will adjust the comment.